### ISSUE TITLE
xcode6.1 iOS8.1

### ISSUE THREAD

**Issue by poholo** [kolyvan/kxmovie#93]
Installing the gas-preprocessor to /usr/bin
Executing chmod +x /usr/bin/gas-preprocessor.pl
Executing cd FFmpeg; ./configure --disable-ffmpeg --disable-ffplay --disable-ffserver --disable-ffprobe --disable-doc --disable-bzlib --target-os=darwin --enable-cross-compile --enable-version3 --arch=arm --cpu=cortex-a8 --enable-pic --extra-cflags='-arch armv7 -miphoneos-version-min=6.0' --extra-ldflags='-arch armv7 -miphoneos-version-min=6.0' --extra-cflags='-mfpu=neon -mfloat-abi=softfp' --enable-neon --enable-optimizations --disable-debug --disable-armv5te --disable-armv6 --disable-armv6t2 --enable-small --cc=/Applications/XCode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang  --sysroot=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS7.1.sdk 
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.10.sdk/usr/include/c++/4.2.1
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.10.sdk/usr/include/c++/4.2.1
/Applications/XCode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang is unable to create an executable file.
C compiler test failed.

If you think configure made a mistake, make sure you are using the latest
version from Git.  If the latest version fails, report the problem to the
ffmpeg-user@ffmpeg.org mailing list or IRC #ffmpeg on irc.freenode.net.
Include the log file "config.log" produced by configure as this will help
solving the problem.
rake aborted!
*******\* Build failed ********
/Users/mac/Desktop/thirdPro/github/kxmovie/Rakefile:8:in `system_or_exit'
/Users/mac/Desktop/thirdPro/github/kxmovie/Rakefile:159:in`buildArch'
/Users/mac/Desktop/thirdPro/github/kxmovie/Rakefile:207:in `block in <top (required)>'
Tasks: TOP => default => build_ffmpeg => build_ffmpeg_armv7
(See full trace by running task with --trace)

**Comment by noyau09** [comment]
hi poholo, did you fix it? I am troubled by the same issue.

**Comment by davieds** [comment]
yeah . i resolved  do u any questions

发自我的 iPhone

**Comment by yuchuanfeng** [comment]
你好, 我 rake 后为什么找不到 output 文件夹, 也没有 libkxmovie.a 这个库, 你能解决这个问题吗?  谢谢!

**Comment by wangyandongnx** [comment]
iOS SDK version number is hardcoded to 7.1 in the Rakefile. You need to set the SDK version number to the one matching your system. For example, I have Xcode 6.2 with iOS SDK 8.2 installed on my system. And I need to change SDK_VERSION='7.1' to SDK_VERSION='8.2' in the Rakefile to make it work.

**Comment by wangyandongnx** [comment]
> 你好, 我 rake 后为什么找不到 output 文件夹, 也没有 libkxmovie.a 这个库, 你能解决这个问题吗? 谢谢!

Rake runs only `build_ffmpeg` task by default. You need to run `rake build_all` or `rake build_movie` to get  libkxmovie.a.

**Comment by yuchuanfeng** [comment]
Yes, Thank your reply, Now I have solved  this question, Not use libkxmovie.a file but my own project that contain kxmovie build perfect. thank you.
